.ci/Jenkinsfile

```diff
@@ -279,3 +314,33 @@ def runReleaseWithKeychain() {
     sh(label: 'release-ci.sh', script: '.ci/scripts/release-ci.sh')
   }
 }
+
+def withAzureSigningCreds(Closure body) {
+  def syntheticsRecorderWindowsVault = 'secret/observability-team/ci/synthetics-recorder-windows-vault'
+  def vault = getVaultSecret(secret: syntheticsRecorderWindowsVault).data
+  if (!vault.containsKey('vault_secret_id') || !vault.containsKey('vault_role_id') || !vault.containsKey('vault_addr')) {
+    error("withAzureSigningCreds: ${syntheticsRecorderWindowsVault} secert must contain vault_secret_id, vault_role_id and vault_addr entries")
+  }
+  def token = getVaultSecret.getVaultToken(vault.vault_addr, vault.vault_role_id, vault.vault_secret_id)  
+  def cloudHsmPath = 'release/cloud-hsm'
+  def azureSecrets = getVaultSecret.getVaultSecretObject(vault.vault_addr, 'secret/release/cloud-hsm', token).data
```

Comment:
```suggestion
  def cloudHsmPath = 'secret/release/cloud-hsm'
  def azureSecrets = getVaultSecret.getVaultSecretObject(vault.vault_addr, cloudHsmPath, token).data
```

Reply:
@v1v Are you sure this won't break the lookup down on L328?

Reply:
Ah, derp. My fault. I didn't see that L328 was actually just an error message. Ignore me.